README.md

```diff
@@ -55,6 +55,17 @@ $ docker compose down -v
 
 _Note:_ _The -v is stands for volumes, and will remove all named volumes specified in the docker compose. In this case it will delete the elasticsearch volume with all the saved data._
 
+### Basics of how to use Kibana
+1. Go to the Kibana Web UI at `http://localhost:5601/`, and login with the username `elastic` and the ELASTIC_PASSWORD defined in the .env file. 
+2. Kibana has many ways to view the log data. Start by going to `Discover`, under the Analytics tab from the sidebar. This will prompt you to make a data view. 
+3. Create a new data view. You should see the `logs-generic-default` Data stream which is the logs sent by logstash. 
+4. The index pattern specifies what data the view should use. Set the index pattern to `logs-generic-default`, give it a name and save the data view to Kibana. 
+5. In the discover view you should see logs being received in the graph, and you can edit the time you want to see in the top right. You can expand documents to view all the info from that log. In the list of available fields you can find things like `path` and `response code` which logstash parsed. 
```

Comment:
one thing that is cool is to find the message field and add it as a column, then the logs are much more readable. i remember i've done it in the past, but i never remember well how to do it. i can show it in the class though!